tools.js

```diff
@@ -130,20 +173,19 @@ module.exports = (function () {
 			});
 		}
 	}
+
 	Tools.loadMods = function () {
```

Comment:
Hmm, I guess...
I intend to replace this function by this other though https://github.com/Slayer95/Pokemon-Showdown/blob/tools-split/tools.js#L913-L979 (which allows to opt-out of Tools data load by removing the ``info.js`` plugin, and to use Tools as... tools)

Inclusion of this change here is mainly to allow a subsequent cleaner diff